pulsar-broker/pom.xml

```diff
@@ -425,6 +425,25 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.gaul</groupId>
+        <artifactId>modernizer-maven-plugin</artifactId>
+        <configuration>
+          <failOnViolations>false</failOnViolations>
+          <javaVersion>8</javaVersion>
```

Comment:
I think broker can use `${pulsar.broker.compiler.release}` which is effectively 17.